src/Icon/Fields/Partials/Icon.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Bloom\Modules\Icon\Fields\Partials;
+
+use Illuminate\Filesystem\Filesystem;
+use Log1x\AcfComposer\Builder;
+use Log1x\AcfComposer\Partial;
+
+class Icon extends Partial
+{
+    /**
+     * The partial field group.
+     */
+    public function fields(): Builder
+    {
+        $icon = Builder::make('icon');
+
+        $icon
+            ->addSelect('icon', [
+                'label' => 'Icon',
+                'instructions' => 'Select an icon',
+                'choices' => $this->getIcons(),
+                'allow_null' => 0,
+                'ui' => 1,
+                'ajax' => 1,
+            ]);
+
+        return $icon;
+    }
+
+    protected function getIcons(): array
+    {
+        $filesystem = new Filesystem();
+        $files = $filesystem->allFiles(get_template_directory() . '/resources/images/icons');
```

Comment:
Would it make sense to use a Bloom config, and set this in there?

I can foresee an easier time updating repositories in future as Bloom evolves if we have per-app configuration for this, rather than putting it in separate files.

We have a config directory already, is it possible to use that?

Reply:
@Jamiewarb Are you referring to the file path specifically? If so, yeah that makes sense 💪